heron/executor/src/python/heron_executor.py

```diff
@@ -155,7 +171,14 @@ def init_parsed_args(self, args):
     # id within docker, rather than the host's hostname. NOTE: this 'HOST' env variable is not
     # guaranteed to be set in all Docker executor environments (outside of Marathon)
     if is_docker_environment():
-      self.master_host = os.environ.get('HOST') if 'HOST' in os.environ else socket.gethostname()
+       # Need to set the HOST environment vairable if docker is for AWS ECS tasks
```

Comment:
Instead of adding code to this function, can we make a overall function called get_host - in which we can check whether it is running in docker or ECS or use socket.gethostname() and return the hostname.